sdk/communication/communication-identity/test.env

```diff
@@ -0,0 +1,3 @@
+# Our tests assume that TEST_MODE is "playback" by default. You can
```

Comment:
the `COMMUNICATION_CONNECTION_STRING` example should be in here too since it is used in tests